packages/components/src/core/Accordion/components/AccordionHeader/index.tsx

```diff
@@ -4,18 +4,19 @@ import Icon from "src/core/Icon";
 
 export interface SdsAccordionHeaderProps {
   subtitle?: string;
+  chevronSize?: "xs" | "s";
 }
 
 export type AccordionHeaderProps = RawAccordionSummaryProps &
   SdsAccordionHeaderProps;
 
 const AccordionHeader = (props: AccordionHeaderProps) => {
-  const { children, id, subtitle } = props;
+  const { children, subtitle, chevronSize = "xs" } = props;
+
   return (
     <StyledAccordionHeader
-      aria-controls={`${id}-panel-content`}
```

Comment:
I see that `aria-controls` is removed, is that because it's misused or something in the a11y error? Thank you!